src/locales/cs.json

```diff
@@ -0,0 +1,1128 @@
+{
```

Comment:
This language is garbage and should be deprecated.

Reply:
I agree